transaction/src/lib.rs

```diff
@@ -8,10 +8,12 @@ mod witness_data;
 
 pub mod action;
 pub mod plan;
+pub mod transaction_view;
```

Comment:
Since this is already in the `transaction` crate, it'd be better to call the module `view`, so that instead of `penumbra_transaction::transaction_view`, we have `penumbra_transaction::view`.